.pr-preview.json

```diff
@@ -1,4 +1,4 @@
 {
-    "src_file": "index.html",
+    "src_file": "\\docs\\virtualkeyboard\\index.html",
```

Comment:
are those Windows-slashes?

Reply:
Thanks! fixed. This is probably why the preview functionality didn't work :)